index.md

```diff
@@ -3,21 +3,23 @@ layout: default
 title: SapMachine
 ---
 
-**SapMachine** is a free, open-source, non-commercial, cross-platform, and fully-functional production-grade version of the Open Java Development Kit (OpenJDK).
-Built and supported by SAP, it provides a robust and reliable Java environment.
+**SapMachine** is a distribution of [OpenJDK](https://openjdk.org/) developed by <a href="https://sap.com">SAP</a>. It is designed to be free, cross-platform, production-ready, and [Java SE certified](https://github.com/SAP/SapMachine/wiki/Certification-and-Java-Compatibility). This distribution serves as the default Java Runtime for SAP's numerous applications and services.
 
 <img align="left" width="240" src="assets/images/logo_circular.svg" alt="Logo of SapMachine">
 
-Following the release cadence of the OpenJDK, a new feature release is shipped every six months, with one designated as a long-term support release every two years.
-Update releases for active versions are provided every quarter. So SapMachine offers long-term support, regular performance improvements and timely security updates.
+This distribution supports all major operating systems.
+SapMachine comes with long-term support releases that include bug fixes and performance updates; you can learn more about support and maintenance in our [wiki](https://github.com/SAP/SapMachine/wiki/Maintenance-and-Support).
 
-For SAP, the default JDK is SapMachine. It is the engine of countless applications and services, in cloud deployments as well as standalone on-premise installations at both, SAP and its customers.
+Our goal is to keep SapMachine as close to OpenJDK as possible,
+only adding additional features when absolutely necessary; you can find a list of these in the [wiki](https://github.com/SAP/SapMachine/wiki/Differences-between-SapMachine-and-OpenJDK).
 
-SapMachine is available on a variety of Operating System/CPU architecture combinations, and it is certified to adhere to the Java SE standard.
-It can serve as a drop-in replacement for any JDK in Java-based applications, ranging from small desktop applications to high-performance, large-scale server applications.
+Our team's many contributions to the OpenJDK and the ecosystem include the [PowerPC/AIX support](http://openjdk.java.net/projects/ppc-aix-port/), [elastic Metaspace](https://openjdk.org/jeps/387),
+ [helpful NullPointerExceptions](https://openjdk.org/jeps/358), a [website for JFR events](https://sap.github.io/SapMachine/jfrevents/) and [more](https://github.com/SAP/SapMachine/wiki/).
```

Comment:
The last link (more) should be https://github.com/SAP/SapMachine/wiki/Features-Contributed-by-SAP

Reply:
No. The page is outdated. We should only link to it when it is maintained again.